plugins/deals.py

```diff
@@ -0,0 +1,32 @@
+from cloudbot import hook
+from cloudbot.util import web
+import feedparser
+
+@hook.command('meh', autohelp=False)
+def meh():
+	'''List the current meh.com deal.'''
+	url = "https://meh.com/deals.rss"
+
+	feed = feedparser.parse(url)
+	title = feed.entries[0].title
+	link = web.try_shorten(feed.entries[0].link)
+
+	return "meh.com: {} ({})".format(title, link)
+
+@hook.command('slickdeals', autohelp=False)
+def slickdeals():
+	'''List the top 3 frontpage slickdeals.net deals.'''
```

Comment:
Docstrings should use triple quotes, not triple single-quotes.
Command docstrings should also begin with `- ` if they do not specify any arguments, due to how the core generates the help response